### PR TITLE
CHANGELOG update: GCP secrets WAL issue fix

### PR DESCRIPTION
**Pull request by Valarissa**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ initialized=true when a Raft join is in progress. [[GH-10498](https://github.com
 * secrets/database/influxdb: Fix issue where not all errors from InfluxDB were being handled [[GH-10384](https://github.com/hashicorp/vault/pull/10384)]
 * secrets/database/mysql: Fixes issue where the DisplayName within generated usernames was the incorrect length [[GH-10433](https://github.com/hashicorp/vault/pull/10433)]
 * secrets/database: Sanitize `private_key` field when reading database plugin config [[GH-10416](https://github.com/hashicorp/vault/pull/10416)]
+* secrets/gcp: Fix issue with account and iam_policy roleset WALs not being removed after attempts when GCP project no longer exists [[GH-10759](https://github.com/hashicorp/vault/pull/10759)]
```
Comment (pcman312) [inline CHANGELOG.md:31]: Should this be in https://github.com/hashicorp/vault/tree/master/changelog ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -43,7 +44,7 @@ initialized=true when a Raft join is in progress. [[GH-10498](https://github.com
 SECURITY:
 
 * LDAP Auth Method: We addressed an issue where error messages returned by the
-  LDAP auth methold allowed user enumeration [[GH-10537](https://github.com/hashicorp/vault/pull/10537)]. This vulnerability affects Vault OSS and Vault 
+  LDAP auth method allowed user enumeration [[GH-10537](https://github.com/hashicorp/vault/pull/10537)]. This vulnerability affects Vault OSS and Vault
```
Comment (pcman312) [inline CHANGELOG.md:47]: Same question re: https://github.com/hashicorp/vault/blob/release/1.6.x/changelog/10537.txt

Though for this one 1.6.1 is already released so I'm unsure of the correct approach here.

Comment (Valarissa) [reply CHANGELOG.md:47]: I didn't create that one, but, sure, I can add it. I didn't even know this changelog folder was a thing...

**Comment by calvn** [comment]
The entry _might_ be overwritten by go-changelog when that's auto-gen'ed. @mladlow is it fine to append entries like this directly or should it be always though a CL file?

**Comment by mladlow** [comment]
The note in 1.6.1 is good - will not be overridden, and is how we need to update old changelogs. The one for 1.7 will need a changelog file, it will be overridden when I do the changelog for 1.7. @Valarissa there's a wiki page describing the changelog process, and you probably have a link to the RFC in your email.

**Comment by Valarissa** [comment]
Just updated this one.

File: CHANGELOG.md (hunk 1 above)
Comment (calvn) [reply CHANGELOG.md:31]: We can remove this entry now that it's in `changelog/10759.txt`.

**Review by calvn — APPROVED** [review]
Small comment, otherwise looks good!

**Review by calvn — APPROVED** [review]

**Pull request merged**